crates/winscard/Cargo.toml

```diff
@@ -11,6 +11,7 @@ description = "A Rust implementation of WinSCard"
 
 [features]
 std = ["tracing/std", "base64", "picky-asn1-der"]
+scard = ["dep:flate2"]
```

Comment:
What is the point of the `scard` feature for the `winscard` crate?
I mean, if the user uses the `winscard` crate, then, obviously, the scard is performed.